src/proto/ball.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
+
+package TbotsProto;
+
+import "proto/vision.proto";
+import "proto/geometry.proto";
+import "proto/tbots_timestamp_msg.proto";
+
+message Ball
+{
+    required BallState current_state = 1;
+    required Vector acceleration     = 2;
```

Comment:
we don't really use acceleration right now, so I think we should remove it and add it later if we actually end up using it

Reply:
Removed.